Source/Core/VideoBackends/D3D/main.cpp

```diff
@@ -87,6 +87,25 @@ void VideoBackend::FillBackendInfo()
 
   g_Config.backend_info.Adapters = D3DCommon::GetAdapterNames();
   g_Config.backend_info.AAModes = D3D::GetAAModes(g_Config.iAdapter);
+
+  // Override optional features if we are actually booting.
+  if (D3D::device)
+  {
+    g_Config.backend_info.bSupportsST3CTextures =
```

Comment:
Typo :(

Reply:
Could you be more specific?

Reply:
it's bSupportsST3CTextures i think it should be bSupportsS3TCTextures?